### ISSUE TITLE
Serve sealed-secrets-web with path based ingress

### ISSUE THREAD

**Issue by dorroddorrod** [ricoberger/sealed-secrets-web#11]
Hi,
I'm trying to implement this web-ui on my EKS cluster and expose it through Ingress.
I'm using Nginx ingress and working with path based routing (e.g example.com/sealed-secrets)

This is how my ingress object looks like:
```yaml
apiVersion: extensions/v1beta1
kind: Ingress
metadata:
  annotations:
    kubernetes.io/ingress.class: nginx
    nginx.ingress.kubernetes.io/force-ssl-redirect: 'true'
    nginx.ingress.kubernetes.io/rewrite-target: /$1
    nginx.ingress.kubernetes.io/ssl-passthrough: 'true'
  name: sealed-secrets-web
  namespace: kube-system
spec:
  rules:
    - host: example.com
      http:
        paths:
          - backend:
              serviceName: sealed-secrets-web
              servicePort: http
            path: /sealed-secrets/?(.*)
```

When i try to render the web, the static content path is missing the `/sealed-secrets` and cannot be found so i'm getting 404.

![Screen Shot 2021-04-24 at 10 12 57](https://user-images.githubusercontent.com/31075689/115950774-e4eaef00-a4e5-11eb-8c7b-0abd7d576be8.png)

Any idea how can i make it work with path based routing?

**Comment by ricoberger** [comment]
Hi @dorroddorrod, can you check if the file is available at `https://yourdomain.com/sealed-secrets/static/ace/ace.js`, if this is the case we can:

1. Remove the remove the leading `/` for the `/static/` routes in the [index.html](https://github.com/ricoberger/sealed-secrets-web/blob/master/static/index.html)  file.
2. Or we can add an option to set a prefix, which is rendered in the template.

I think the second option would be the better one and should be the same as used by other tools like Prometheus for their UI.

**Comment by dorroddorrod** [comment]
Hi @ricoberger ,
Thanks for the fast response,
`https://yourdomain.com/sealed-secrets/static/ace/ace.js` is available. 
![Screen Shot 2021-04-25 at 20 47 05](https://user-images.githubusercontent.com/31075689/116003693-b2391780-a607-11eb-8943-294cf9a02b60.png)

Option number 2 indeed sounds better.

**Comment by ricoberger** [comment]
Hi @dorroddorrod, I published a new version [2.2.1](https://github.com/ricoberger/sealed-secrets-web/releases/tag/2.2.1), which adds a flag `--web-external-url`.

Please let me know, if you still have problems.

**Comment by dorroddorrod** [comment]
Awesome !!!! thank you very much !
